<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(81, 81, 77);
color:white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ExitButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>850</y>
      <width>171</width>
      <height>151</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="statusTip">
     <string notr="true"/>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
    <property name="accessibleName">
     <string notr="true"/>
    </property>
    <property name="accessibleDescription">
     <string notr="true"/>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset;
font: 16pt &quot;Free Helvetian&quot;;
color:black;
 text-align: center;
 background-color: rgb(231, 126, 35);
</string>
    </property>
    <property name="text">
     <string>ВЫХОД</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>171</width>
      <height>791</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="statusTip">
     <string notr="true"/>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
    <property name="accessibleName">
     <string notr="true"/>
    </property>
    <property name="accessibleDescription">
     <string notr="true"/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(194, 194, 194);
color:black;
font: 14pt &quot;Free Helvetian&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="iconSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideLeft</enum>
    </property>
    <property name="flow">
     <enum>QListView::TopToBottom</enum>
    </property>
    <property name="isWrapping" stdset="0">
     <bool>false</bool>
    </property>
    <property name="layoutMode">
     <enum>QListView::Batched</enum>
    </property>
    <property name="gridSize">
     <size>
      <width>0</width>
      <height>60</height>
     </size>
    </property>
    <property name="viewMode">
     <enum>QListView::ListMode</enum>
    </property>
    <property name="batchSize">
     <number>30</number>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="selectionRectVisible">
     <bool>false</bool>
    </property>
    <property name="currentRow">
     <number>-1</number>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>1081</width>
      <height>991</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string notr="true"/>
    </property>
    <property name="statusTip">
     <string notr="true"/>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
    <property name="accessibleName">
     <string notr="true"/>
    </property>
    <property name="accessibleDescription">
     <string notr="true"/>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(194, 194, 194);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(194, 194, 194);
color:black;
font: 14pt &quot;Free Helvetian&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;СПИСОК&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
